cashu/wallet/cli/cli.py

```diff
@@ -444,6 +480,47 @@ async def burn(ctx: Context, token: str, all: bool, force: bool, delete: str):
     wallet.status()
 
 
+@cli.command("restore", help="Restore backups.")
+@click.option(
+    "--to",
+    "-t",
+    default=2,
+    help="Split minted tokens with a specific amount.",
+    type=int,
+)
```

Comment:
Maybe it's just me being not too deep into the code but it isn't quite clear to me what this `to` option / variable is doing.  Also at the other places where it is used (I think router.py and wallet.py). 

Could we maybe improve the name or the docstrings a bit to make it better understandable what this is doing?

Reply:
Oh yeah, nice catch. The docstring is totally wrong. This flag specifies how many batches should be attempted to be restored after no new proof is found. I think users should never make use of this (the default value should just work) but I left it in for experimenting 
```suggestion
    default=2,
    help="Split minted tokens with a specific amount.",
    type=int,
)
@click.option(
    "--batch",
    "-b",
```until we figure out what the best parameters for this should be.